src/vs/workbench/parts/preferences/electron-browser/settingsEditor2.ts

```diff
@@ -837,6 +857,12 @@ export class SettingsEditor2 extends BaseEditor {
 
 		resolvedSettingsRoot.children.push(resolveExtensionsSettings(dividedGroups.extension || []));
 
+		// Make sure that all extensions' settings are included in search results
+		const cachedState = this.editorMemento.loadState(this.group, this.input);
+		if (cachedState && cachedState.searchQuery) {
+			this.triggerSearch(cachedState.searchQuery);
```

Comment:
I'm not totally sure, but since this will trigger a render when it's done, can we avoid calling renderTree here?

Also, is this always safe to call before settingsTreeModel has been created?